src/snippets/switch.md

```diff
@@ -0,0 +1,57 @@
+---
+description: Switch Statement Snippet
+---
+
+# Switch
+
+Using `switch` statements in AssemblyScript
+
+```editor
+#!optimize=size&runtime=full
+export function switchSurprise(a: i32): i32 {
+  let response = -1;
+  switch(a) {
```

Comment:
```suggestion
  switch (a) {
```

Also, typical code style on the docs is omitting semicolons since it is a bit easier to read and often considered "more modern". Differs from the compiler, where the greater goal is to be better safe than sorry :)